denys-kravchuk/README.md

```diff
@@ -0,0 +1,73 @@
+# Social network for volunteers
+
+# Description
+This social network was created to help our volunteers and unite them on the one platform.
+
+# Presentation
+![chrome_0Z5R5Betcu](https://github.com/Denys-Kravchuk982910/op-kp23-Kravchuk/assets/114987963/0ab54bd1-dc09-4d57-9382-fa366ee5343d)
+
+![chrome_YgeWL5Y9A8](https://github.com/Denys-Kravchuk982910/op-kp23-Kravchuk/assets/114987963/1cb11d40-19f3-4a67-ae20-080e5ff75ab8)
+
+![chrome_S6wQlsydWS](https://github.com/Denys-Kravchuk982910/op-kp23-Kravchuk/assets/114987963/71e1498a-9125-4cf3-9567-be2b8a36ce07)
+
+
+
+# Get started
+1. Clone the server part: git clone https://github.com/Denys209818/DiplomaWork.git
+2. Clone the client part: https://github.com/Denys209818/DiplomaWorkFront.git
+3. you should publish these projects using embedded vs functions or just use special commands dotnet publish, npm publish.
```

Comment:
- Use capital case in the beginning of a sentence: You
- Continue the list consistently: 1. Clone... 2. Clone 3. Publish... and so on.
- Use monospaced font for commands with Markdown backticks: `dotnet publish`, `npm publish`.